lib/mongoid/config.rb

```diff
@@ -73,6 +73,7 @@ module Config
 
     # Return stored times as UTC.
     option :use_utc, default: false
+    option :map_big_decimal_to_decimal128, default: false
```

Comment:
This line needs an empty line above it to separate this option from the previous option.

Reply:
done